src/server.ts

```diff
@@ -242,9 +242,9 @@ export class Server {
    * To fetch contract wasm byte-code, use the ContractCode ledger entry key.
    *
    * @example
-   * const contractId = "0000000000000000000000000000000000000000000000000000000000000001";
+   * const contractId = "CAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD2KM";
```

Comment:
minor, I noticed similar examples with the hex id string usage in js-docs of other methods in server.ts like[ getLedgerEntries() ](https://github.com/stellar/js-soroban-client/blob/421d2bf5ef0fd8b0ff08b2508e4ceb21693d485c/src/server.ts#L404-L405), is it worthwhile for consistency to give those the same update?

Reply:
Thought I got em all but my `Ctrl+F`-fu wasn't great. Thanks for pointing this out! Should've got 'em all, now.